src/js/background/assignManager.js

```diff
@@ -274,6 +306,38 @@ const assignManager = {
     }
   },
 
+  async _onClickedBookmark(info) {
+
+    async function _getBookmarksFromInfo(info) {
+      const [bookmarkTreeNode] = await browser.bookmarks.get(info.bookmarkId);
+      if (bookmarkTreeNode.type === "folder") {
+        return await browser.bookmarks.getChildren(bookmarkTreeNode.id);
+      }
+      return [bookmarkTreeNode];
+    }
+
+    const bookmarks = await _getBookmarksFromInfo(info);
+    for (const bookmark of bookmarks) {
+      // Some checks on the urls from https://github.com/Rob--W/bookmark-container-tab/ thanks!
+      if ( !/^(javascript|place):/i.test(bookmark.url) && bookmark.type !== "folder") {
+        const openInReaderMode = bookmark.url.startsWith("about:reader");
+        if(openInReaderMode) {
+          try {
+            const parsed = new URL(bookmark.url);
+            bookmark.url = parsed.searchParams.get("url") + parsed.hash; // can't believe const lets me do this ...
+          } catch (err) {
+            return err.message;
+          }
+        }
+        browser.tabs.create({
```

Comment:
> Oh dang. This means nothing will happen when a user tries to open a `javascript:` or `place:` bookmark into a container, with no error message nor alt text telling them why they can't.
> 
> If we get bug reports about it, we could try moving this logic "up" into `init` before we add the listener to the context menu item.

For this we can just add a callback here. If the returned promise throws an error and a new tab cannot be created then generate an alert message for the same.